components/trace/appmap/index.mjs

```diff
@@ -19,6 +29,36 @@ const { orderEventArray } = await import(`./ordering/index.mjs${__search}`);
 
 const VERSION = "1.8.0";
 
+const summary_template =
+  "Received %j raw events.\n\nEvent Repartition:\n%f\n\nApply Repartition:\n%f\n";
```

Comment:
I don't think "repartition" is the right word. Nothing is being *re*distributed, is it? Seems like you just mean "distribution".

Regardless, it can still be difficult to understand. Consider saying something like "most frequently recorded events" (if that's what it is).

Reply:
Done https://github.com/getappmap/appmap-agent-js/commit/c82ff52d19b010d273ed32a1fc14a0393d22f492